.github/workflows/rbc_test.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Do we want to keep it always `false`?

Reply:
I think it is better for debugging when fail-fast is false.